api/metric/must.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+// MeterMust is a wrapper for Meter interfaces that panics when any
+// instrument constructor encounters an error.  It implements the
+// `MeterMethods` interface but the signature of the constructors is
+// different.
+type MeterMust struct {
+	meter Meter
+}
+
+// Must constructs a MeterMust implementation from a Meter, allowing
+// the application to panic when any instrument constructor yields an
+// error.
+func Must(meter Meter) MeterMust {
+	return MeterMust{meter: meter}
+}
+
+// MeasureConstructorsMust are part of the `MeterMust` API, a Meter
+// where `New` methods panic instead of return an error.
+type MeasureConstructorsMust interface {
```

Comment:
I don't think these are needed anymore with then change to `MeterMust` being a struct now, right?

Reply:
Already removed.